Formula/terminator.rb

```diff
@@ -0,0 +1,42 @@
+class Terminator < Formula
+  include Language::Python::Virtualenv
+
+  desc "Multiple GNOME terminals in one window"
+  homepage "https://gnome-terminator.org"
+  url "https://github.com/gnome-terminator/terminator/archive/refs/tags/v2.1.1.tar.gz"
+  sha256 "ee1907bc9bfe03244f6d8074b214ef1638a964b38e21ca2ad4cca993d0c1d31e"
+  license "GPL-2.0-only"
+
+  depends_on "dbus"
+  depends_on "pygobject3"
+  depends_on "python@3.9"
+  depends_on "vte3"
+
+  resource "dbus-python" do
+    url "https://files.pythonhosted.org/packages/62/7e/d4fb56a1695fa65da0c8d3071855fa5408447b913c58c01933c2f81a269a/dbus-python-1.2.16.tar.gz"
+    sha256 "11238f1d86c995d8aed2e22f04a1e3779f0d70e587caffeab4857f3c662ed5a4"
+  end
+
+  resource "psutil" do
+    url "https://files.pythonhosted.org/packages/e1/b0/7276de53321c12981717490516b7e612364f2cb372ee8901bd4a66a000d7/psutil-5.8.0.tar.gz"
+    sha256 "0c9ccb99ab76025f2f0bbecf341d4656e9c1351db8cc8a03ccd62e318ab4b5c6"
+  end
+
+  def install
+    # virtualenv_install_with_resources
+    venv = virtualenv_create(libexec, "python3")
+    %w[dbus-python psutil].each do |r|
+      venv.pip_install resource(r)
+    end
+    venv.pip_install_and_link buildpath
+
+    system "#{Formula["python@3.9"].opt_bin}/python3", "setup.py", "build"
+    system "#{Formula["python@3.9"].opt_bin}/python3", "setup.py", "install", "--prefix=#{prefix}"
```

Comment:
Are you sure this is needed? I think this just duplicates `venv.pip_install_and_link buildpath`.

Reply:
not needed, fixed with the new commit